backend/src/channel/channel.service.ts

```diff
@@ -40,6 +41,54 @@ export class ChannelService {
   ) {}
   logger: Logger = new Logger('ChannelService');
 
+  /**
+   * @summary 채널 목록 조회하기
+   */
+  getChannelsList(cursor: number): ChannelsListResponseDto {
+    const channels = this.visibleChannelRepository.findByCursor(cursor).map(({ id, name, mode, users }) => {
+      return { id, name, mode, count: users.size };
+    });
+    return {
+      total: cursor === 0 ? this.visibleChannelRepository.count() : undefined,
+      channels,
+    };
+  }
+
+  /**
+   * @summary 채널 정보 조회하기
+   */
+  getChannelInfo(myId: number, channel: Channel): ChannelMemberInfoResponseDto {
+    const channelUser = channel.users.get(myId);
+    if (channelUser === undefined) {
+      throw new NotFoundException('해당 채널에 참여중인 유저가 아닙니다.');
+    }
+    const users = [...channel.users.values()];
+    const players: MemberInfo[] = users
+      .filter((user) => user.isPlayer)
+      .map((user) => {
+        return {
+          userId: user.id,
+          nickname: user.nickname,
+          image: user.image,
+          role: user.role,
+        };
+      });
+    const observers: MemberInfo[] = users
+      .filter((user) => !user.isPlayer)
+      .map((user) => {
+        return {
+          userId: user.id,
+          nickname: user.nickname,
+          image: user.image,
+          role: user.role,
+        };
+      });
+    return {
+      players: players,
+      observers: observers,
+    };
```

Comment:
```suggestion
    return { 
      players,
      observers 
    };
```

제안드립니닷~ 한 줄로도 되면 좋은데 prettier 규칙 있을 것 같네요